bundles/io/org.eclipse.smarthome.io.rest.log/README.md

```diff
@@ -0,0 +1,119 @@
+FORMAT: 1A
+
+# Eclipse SmartHome UI Logging Bundle
+
+This Bundle provides a resource at the REST API to enable a consumer to fetch the set log levels.
+Additionally it receives log messages which will be logged to the server's log. 
+The last logged messages can be requested to be displayed in other UIs.
+
+Meta: This document is in normal markdown format, 
+but also compatible to [Apiary](apiary.com) for automatic doc generation and API testing.
+
+## Log levels [/rest/log/levels]
+
+### Get enabled log levels [GET]
+
```

Comment:
I think this feature is not about allowing to configure your server logging, but to allow UIs to log messages to the server side log. So GET should be fine.